### PR TITLE
Ai Added

### LINKED ISSUE #1225: Azure Chat bot by Microsoft
**Is your feature request related to a problem? Please describe.**
Implementing azure chatbot help user to get to know more about 

**Describe the solution you'd like**
User Friendly , Helping hand

### PR DESCRIPTION
**Pull request by dhruv8433**
# Description

@thestarsahil 

Azure Chat bot by Microsoft #1225

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #1225(issue_no)

Replace `issue_no` with the issue number which is fixed in this PR

## Screenshots

https://github.com/user-attachments/assets/459211d8-75d1-4b8c-96b5-7477828c022a

## Checklist

- [x] Tests have been added or updated to cover the changes
- [x] Documentation has been updated to reflect the changes
- [x] Code follows the established coding style guidelines
- [x] All tests are passing

### CODE REVIEW

**Comment by dhruv8433** [comment]
@thestarsahil 

kindly please check this and merge it

**Comment by thestarsahil** [comment]
Dear that's a reference not to copy and paste 
UI , Proper responsive need to work on

**Comment by dhruv8433** [comment]
@thestarsahil 

please check this 
i added fully functional chatbot 

https://github.com/user-attachments/assets/62c4e5f5-e8c7-4376-92f7-b8ac3528015c

**Comment by thestarsahil** [comment]
from where the data fetching ?

**Comment by dhruv8433** [comment]
@thestarsahil 

I am using botPengiun where we can configure out ai chatbot for our website according our requirement

https://botpenguin.com/

**Comment by dhruv8433** [comment]
@thestarsahil 

please review it and if you need any modification please inform me

**Comment by thestarsahil** [comment]
@dhruv8433 We are implementing Azure services in this project

**Comment by dhruv8433** [comment]
@thestarsahil 

check this i implemented Azure chatbot and make it more attractive by adding styles

https://github.com/user-attachments/assets/5775cbae-2467-4218-8e33-92ab5656fbd0

please check this and merge it

**Comment by dhruv8433** [comment]
@thestarsahil 

please sir review it and merge ir

**Comment by thestarsahil** [comment]
Packages not uploaded properly

**Comment by thestarsahil** [comment]
Screenshot from 2024-08-05 16-52-44

**Comment by dhruv8433** [comment]
@thestarsahil 

please check now

image

**Comment by thestarsahil** [comment]
Chatbot not should be login page 
Screenshot from 2024-08-05 17-09-16

On First Time its showing this 
Screenshot from 2024-08-05 17-09-33

On Closing and reopeing again its showing this 
Screenshot from 2024-08-05 17-09-42

**Comment by dhruv8433** [comment]
@thestarsahil 

please check now

https://github.com/user-attachments/assets/58431164-a4d8-4a1d-920d-f9aae49e6de1

**Comment by dhruv8433** [comment]
@thestarsahil 

when you become free please review it and merge it
also add level as per #1225

**Comment by thestarsahil** [comment]
Resolve conflict and push again

**Comment by dhruv8433** [comment]
@thestarsahil 

now you can merge it

**Comment by dhruv8433** [comment]
@thestarsahil

when you become free please review it and merge it
also add level as per https://github.com/Counselllor/Counsellor-Web/issues/1225

**Comment by dhruv8433** [comment]
@thestarsahil 

i fetched upstream please sir check this and merge it

**Comment by thestarsahil** [comment]
@dhruv8433 Check the review part i mentioned

**Comment by dhruv8433** [comment]
@thestarsahil 

i didn't get any changes in reviews section 
image

**Review by thestarsahil — CHANGES_REQUESTED** [review]

File: src/components/FAQs/FAQs.jsx
[the remark this answers is not in the document]
Comment (thestarsahil) [reply src/components/FAQs/FAQs.jsx:1]: @dhruv8433

File: src/components/chatbot/Chatbot.jsx
[the remark this answers is not in the document]
Comment (thestarsahil) [reply src/components/chatbot/Chatbot.jsx:1]: @dhruv8433

**Comment by dhruv8433** [comment]
@thestarsahil 

i revert faqs and remove chat bot from login page 

image

image

**Comment by dhruv8433** [comment]
@thestarsahil 

please review it and merge it

**Comment by thestarsahil** [comment]
can you please resolve the problem for the last time

**Comment by dhruv8433** [comment]
@thestarsahil 

what i have to resolve

**Comment by dhruv8433** [comment]
@thestarsahil 

in my local build is success

image

please tell me if you want any modification

**Comment by thestarsahil** [comment]
Screenshot from 2024-08-09 20-30-48

**Comment by dhruv8433** [comment]
@thestarsahil 

can you please try again i try to add upstream package here

**Comment by dhruv8433** [comment]
@thestarsahil 

Please sir merge thi
Today is last day and i am going back....

I completed and reviews package please merge it

**Comment by dhruv8433** [comment]
@thestarsahil 

please merge this

**Comment by thestarsahil** [comment]
@dhruv8433 npm run build is not working on this PR kindly check once

**Pull request closed without merging**